src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.Person;
+import java.util.ArrayList;
+
+public class Tagging {
+	
+	public enum TaggingType {ADD, DELETE};
+	private static final String ADD_SYMBOL = "+";
+	private static final String DELETE_SYMBOL = "-";
+	
+	private TaggingType taggingType;
+	private Person person;
+	private Tag tag;
+	public static ArrayList<Tagging> allTaggings = new ArrayList<Tagging>();
```

Comment:
The `Tagging` list should not be here.  A tagging only associates a person with a tag.  Let `AddressBook` handle the `Tagging` list.